stable/mail2most/templates/pvc.yaml

```diff
@@ -0,0 +1,24 @@
+{{ if .Values.persistence.enabled -}}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "name" . }}-pvc
+  labels:
+    app.kubernetes.io/name:  {{ template "name" . }}
+    app.kubernetes.io/instance: "{{ .Release.Name }}"
+    app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+spec:
+  accessModes:
+    - ReadWriteMany
```

Comment:
in the values you define `ReadWriteOnce` and here is `ReadWriteMany` make it configurable